common-tasks/backup-restore.md

```diff
@@ -65,7 +119,7 @@ Restoring from a Backup
 3. There are three options you may select when restoring from a backup:
    1.  **ignore missing**: If any of the VMs in your backup depended upon a NetVM, ProxyVM, or TemplateVM that is not present in (i.e., "missing from") the current system, checking this box will ignore the fact that they are missing and restore the VMs anyway.
    2.  **ignore username mismatch**: This option applies only to the restoration of dom0's home directory. If your backup was created on a Qubes system which had a different dom0 username than the dom0 username of the current system, then checking this box will ignore the mismatch between the two usernames and proceed to restore the home directory anyway.
-   3.  **skip dom0**: If this box is checked, dom0's home directory will not be restored from your backup.
+   3.  **Verify backup integrity, do not restore the data**: Like it says- check this if you only want to verify.
```

Comment:
It might be worth saying a bit more about what this option does (simulates a full restore of all selected data from the backup without persisting the data on disk).

Reply:
How about this?
This will scan the backup file for corrupted data. However, it does not currently detect if it is missing data, as long as it is a correctly structured, non-corrupted backup file. See [issue #3498](https://github.com/QubesOS/qubes-issues/issues/3498) for more details.

Reply:
Much better!